### PR TITLE
Added Eid register

### PR DESCRIPTION
**Pull request by carapai**

### CODE REVIEW

**Comment by ssmusoke** [comment]
@carapai 

1. Add the EID register to the list of registers

2. The commit should not change the 105 Excel templates 

3. Can you also add a start and end date to the register 

4. Register data:
- Some ages have -1 
- Format the dates to be consistent with other registers
- Merge the cells so as to look like the regsiter, the current order is confusing

**Comment by carapai** [comment]
Please @ssmusoke look at the updated pull request
Also refer to the register documentation for date formats
The -1 in age is a problem with data entry, baby opened chat before birth

File: api/src/main/java/org/openmrs/module/ugandaemrreports/reports/Helper.java [hunk 1]
```diff
@@ -1,6 +1,5 @@
 package org.openmrs.module.ugandaemrreports.reports;
 
-import com.google.common.base.Joiner;
 import com.google.common.base.Splitter;
 import com.google.common.collect.*;
```
Comment (ssmusoke) [inline api/src/main/java/org/openmrs/module/ugandaemrreports/reports/Helper.java]: @carapai can you include the specific libraries not just a wild card import

**Comment by ssmusoke** [comment]
@carapai No data is shown see attached 
[EID_Register_2017-12-06_09_39_34-1.xlsx](https://github.com/METS-Programme/openmrs-module-ugandaemr-reports/files/1534152/EID_Register_2017-12-06_09_39_34-1.xlsx)

**Pull request merged**